docs/FAQ.md

```diff
@@ -506,6 +506,37 @@ Try increasing the
 [`spark.kryoserializer.buffer.max`](https://spark.apache.org/docs/latest/configuration.html#compression-and-serialization)
 from a default of 64M to something larger, for example 512M.
 
+### Why am I getting "Unable to acquire buffer" or "Trying to free an invalid buffer"?
+
+The RAPIDS Accelerator makes use of a tracking system called `RapidsBufferCatalog` for GPU memory
```

Comment:
I think we can simplify this a bit. Not sure we need to explain everything that uses spillable memory. I think the users will be able to infer that part if needed.

```
The RAPIDS Accelerator tracks GPU allocations that can be spilled to CPU memory or even disk using the
`RapidsBufferCatalog` class.
```